tools/wptrunner/wptrunner/browsers/firefox.py

```diff
@@ -248,6 +248,7 @@ def start(self, group_metadata=None, **kwargs):
             "network.proxy.type": 0,
             "places.history.enabled": False,
             "network.preload": True,
+            "dom.file.createInChild": True
```

Comment:
I’m surprised this isn’t set automatically by Marionette:
https://searchfox.org/mozilla-central/source/testing/marionette/components/marionette.js#179

Are you overwriting it somewhere else?

Reply:
We don't use GeckoInstance.

Reply:
That’s not GeckoInstance. We set it as a recommended preference when Firefox starts.

Reply:
Oh, right, sorry but we don't get marionette to start firefox either.

Reply:
Right, but you start Firefox with `--marionette` right? That should initialise the server and set recommended prefs unless `marionette.prefs.recommended` is set to false.

I’m not saying adding this pref is wrong, but I don’t understand that you need it.

Reply:
Ah, the unittest pref set which we apply set that to false: https://searchfox.org/mozilla-central/source/testing/profiles/unittest/user.js#193. Which makes some sense since it makes things a bit less explicit in terms of what's enabled.

Reply:
OK, then I think it makes sense to re-enable it here rather than changing the underlying file.